index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMovies, getMovieByTitle, getMoviesByDirector, createNewMovie } = require('./controllers/movies')
+
+const app = express()
+
+app.get('/', getAllMovies)
+
+app.get('/:title', getMovieByTitle)
+
+app.get('/:director', getMoviesByDirector)
```

Comment:
These are the same route. When the user types `http://localhost:2319/something` Express cannot possibly know whether their intention was a director or a title. It will match the route on line 9 regardless of what the user types and place what they typed into the `title` variable. Line 11 is unreachable code.